src/sentry/features/__init__.py

```diff
@@ -10,6 +10,26 @@
 from .handler import *  # NOQA
 from .manager import *  # NOQA
 
+___all___ = [
+    "add",
+    "entity_features",
+    "get",
+    "has",
+    "batch_has",
+    "all",
+    "add_handler",
+    "add_entity_handler",
+    "has_for_batch",
+    "requires_snuba",
+    "Feature",
+    "FeatureHandlerStrategy",
+    "OrganizationFeature",
+    "ProjectFeature",
+    "ProjectPluginFeature",
+    "SystemFeature",
+    "UserFeature",
+]
```

Comment:
consumers should explicitly import these names.   the `*` imports make several aspects of development _worse_: easier to import cycle, slower test startup, slower app startup, slower mypy execution, worse mypy cacher performance

Reply:
The `__all__` is not only used for `*` imports right? This also allows me to say that this module is re-exporting some of these names, so I can fix errors like this https://github.com/getsentry/sentry/actions/runs/5659967246/job/15334645800?pr=51169#step:5:16

I realize I can also fix this by explicitly importing these from `sentry.features.base`